.hlint.yaml

```diff
@@ -0,0 +1,69 @@
+# HLint configuration file
+# https://github.com/ndmitchell/hlint
+##########################
```

Comment:
Bitte die autogenerierten Kommentare entfernen. Manchmal ist es zwar ganz praktisch, die hier zu lesen, aber in der aktuellen Version findet man die relevanten Zeile im Rauschen eher schlecht.